mkosi/__init__.py

```diff
@@ -815,6 +833,7 @@ def install_unified_kernel(state: MkosiState, roothash: Optional[str]) -> None:
                 *(["--mirror", state.config.mirror] if state.config.mirror else []),
                 "--repository-key-check", yes_no(state.config.repository_key_check),
                 "--repositories", ",".join(state.config.repositories),
+                *flatten([("--package-manager-tree", format_source_target(s, t)) for s, t in state.config.package_manager_trees]),
```

Comment:
This would be more similar to the rest, but I think
```python
*(["--package-manager-tree", ",".join(format_source_target(s, t) for s, t in state.config.package_manager_trees] if state.config.package_manager_trees else [])
```
is less readable than the `flatten`.

Reply:
You should be able to get away with 

```python
"--package-manager-tree", ",".join(...)
```

Reply:
Exchanged it for that.